kanidm_book/src/examples/k8s_ingress_example.md

```diff
@@ -6,69 +6,132 @@ Guard your Kubernetes ingress with Kanidm authentication and authorization.
 
 The following are used in this example:
 
-- [Kanidm](installing_the_server.html)
-- [Kubernetes v1.23](https://docs.k0sproject.io/v1.23.6+k0s.2/install/)
+- [Kanidm](../installing_the_server.html)
+- [Kubernetes v1.23 or above](https://docs.k0sproject.io/v1.23.6+k0s.2/install/)
 - [Nginx Ingress](https://kubernetes.github.io/ingress-nginx/deploy/)
+- [A Domain Name](https://domains.google/)
 - [CertManager - With Cluster Issuer](https://cert-manager.io/docs/installation/)
 - [ShellCheck](https://github.com/koalaman/shellcheck#user-content-installing)
 
 You can set up your certificates manually instead of using CertManager if you are comfortable doing so. If it is your first time setting up CertManager, take your time with installation and validation. For the `http01` solver the ingress port (8089) needs to be accessible from the internet - DNS validation is recommended.
 
 ## Instructions
 
-1. Install the Kubernetes dashboard
-
-```shell
-kubectl create -f https://raw.githubusercontent.com/kubernetes/kops/master/addons/kubernetes-dashboard/v1.10.1.yaml
-```
-
-2. Create a script from the content below, replacing every `<string>` (drop the `<>`) with appropriate values, then run it. This will example users, groups and the OAUTH2 configuration.
+1. Create a script `deplopy.sh` from the content below, replacing every `<string>` (drop the `<>`) with appropriate values, then run it. This will create the user, group and OAUTH2 resource.
 
 ```shell
 #!/bin/sh
 
 # Analysis self for common errors.
 shellcheck ./*.sh || exit 1
 
+# Login
+echo "User: idm_admin"
+kanidm login --name idm_admin
+
 # Create User
-kanidm account create k8s_example_user
+kanidm account create k8s_example_user "K8s Example User" --name idm_admin
+kanidm account credential update k8s_example_user --name idm_admin
```

Comment:
Could this be added? When the script's run you won't know what's going on:

`echo "Updating the credentials for k8s_example_user..."`